src/managers/message-manager.js

```diff
@@ -180,7 +180,7 @@ function updateMessageByInstanceId(instanceId, message) {
 // Added this to avoid errors in the console
 function handleTouchStartEvents(e) {}
 
-function handleGistEvents(e) {
+async function handleGistEvents(e) {
```

Comment:
Does the addEventListener call for this guy need `async` added?

Reply:
We're not doing anything with the result so we should be good.